src/openroot_optimizer/psychrometric_model.py

```
@@ -0,0 +1,93 @@
+    h_in = moist_air_enthalpy_kj_per_kg_da(T_in, w_in)
+    h_out = moist_air_enthalpy_kj_per_kg_da(T_out, w_out)
+    mdot_da = dry_air_mass_flow_kg_s(airflow_m3_s)
+    delta_h_kj_kg = h_in - h_out
+    return max(0.0, mdot_da * delta_h_kj_kg * 1000.0)
+
+def parse_float(x, default=None):
+    try:
+        if x is None or x == "":
+            return default
+        return float(x)
+    except:
```

Ruff (0.16.2)

[error] Do not use bare `except`

(E722)